cmd/kubeadm/app/preflight/checks.go

```diff
@@ -182,6 +182,38 @@ func (fac FileAvailableCheck) Check() (warnings, errors []error) {
 	return nil, errors
 }
 
+// FileContentCheck checks that the given file contains the string Content.
+type FileContentCheck struct {
+	Path    string
+	Content string
+}
+
+func (fcc FileContentCheck) Check() (warnings, errors []error) {
+	errors = []error{}
+	f, err := os.Open(fcc.Path)
+	if err != nil {
+		errors = append(errors, fmt.Errorf("%s does not exists", fcc.Path))
+		return nil, errors
+	}
+
+	lr := io.LimitReader(f, 1)
```

Comment:
I disagree with you, using `ioutil.ReadFile` is not a good solution. What if the file is absolutely huge? If anything, using a larger buffer would make sense. I decided to limit it with `io.LimitReader` because it is expecting to read exactly one byte. If we need to read larger files in the future, that could be changed, but I wouldn't change it now.